src/sentry/api/endpoints/release_threshold_details.py

```diff
@@ -0,0 +1,29 @@
+from django.http import HttpResponse
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import Project
+from sentry.models.release_threshold.releasethreshold import ReleaseThreshold
+
+
+@region_silo_endpoint
+class ReleaseThresholdDetailsEndpoint(ProjectEndpoint):
+    owner: ApiOwner = ApiOwner.ENTERPRISE
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
+    }
+
+    def get(self, request: Request, project: Project, threshold_id: str) -> HttpResponse:
```

Comment:
i am assuming you will have a PUT (and maybe a DELETE), so it might be worth refactoring with a `convert_args` function so you already have the `ReleaseThreshold` object you're getting/modifying/deleting

Reply:
convert_args just allows us to convert the path param (threshold_id) into the actual model instance, correct?